powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/service/behavior/tasks/v3/OperationServiceBehavior.java

```diff
@@ -102,6 +102,7 @@ public OperationDetailResponse createOperation(OperationCreateRequest request) t
         final List<String> applications = request.getApplications();
         final String activationFlag = request.getActivationFlag();
         final String templateName = request.getTemplateName();
+        final Date timestampExpiresRequest = request.getTimestampExpires();
```

Comment:
Validate the timestamp is not in the past.

Reply:
Good idea.